config-templates/redhat/odf-remote/4.10/custom-parameters.json

```diff
@@ -108,23 +130,30 @@
     "displayname": "KMS encryption",
     "name": "kms-encryption",
     "default": "false",
+    "regex": "^(?:tru|fals)e$",
```

Comment:
same here `"regex": "^(?:true|false)$",`

Reply:
Addressed in https://github.com/IBM/ibm-satellite-storage/pull/194/commits/3638379ea315df3f1df332997fb75a5000b13224